Makefile

```diff
@@ -57,7 +57,7 @@ $(toolchain-mac):
 	#cd $@/build/llvm-10.0.1.src/build && make install
 
 	# Build osxcross
-	cp MacOSX10.13.sdk.tar.bz2 $@/tarballs/
```

Comment:
I think newer versions of osxcross produce .xz tarballs. Mine has that extension and this would support all formats.